firstvoices/backend/models/import_jobs.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.auth import get_user_model
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.utils.translation import gettext as _
+
+from backend.models.async_results import BaseJob
+from backend.models.base import BaseSiteContentModel
+from backend.models.constants import (
+    DEFAULT_TITLE_LENGTH,
+    MAX_DESCRIPTION_LENGTH,
+    MAX_NOTE_LENGTH,
+)
+from backend.models.media import File
+from backend.permissions import predicates
+
+
+class ImportJobMode(models.TextChoices):
+    SKIP_DUPLICATES = "skip_duplicates", _("Skip Duplicates")
+    ALLOW_DUPLICATES = "allow_duplicates", _("Allow Duplicates")
+    UPDATE = "update", _("Update")
+
+
+class RowStatus(models.TextChoices):
+    ERROR = "error", _("Error")
+    INVALID = "invalid", _("Invalid")
+    SKIP = "skip", _("Skip")
+    NEW = "new", _("New")
+    UPDATE = "update", _("Update")
+    DELETE = "delete", _("Delete")
+
+
+class ImportJobReport(BaseSiteContentModel):
+    class Meta:
+        verbose_name = _("Import Job Report")
+        verbose_name_plural = _("Import Job Reports")
+        rules_permissions = {
+            "view": predicates.is_language_admin_or_super,
```

Comment:
This is fine for now, but I realized it doesn't match the tech design. Could you please log a ticket in the epic about user self-serve imports to fix the permissions to include editors?

Reply:
Or actually, since the filters are already done, you could just switch these to is_at_least_editor_or_super